src/smt/set_defaults.cpp

```diff
@@ -165,7 +165,11 @@ void setDefaults(LogicInfo& logic, bool isInternalSubsolver)
 
     // Force lazy solver since we don't handle EAGER_ATOMS in the
     // BVSolver::BITBLAST solver.
-    opts.set(options::bvSolver, options::BVSolver::LAZY);
+    if (options::bvSolver() != options::BVSolver::BITBLAST
```

Comment:
Update the comment above.

Reply:
I removed the code block since all BV solver now support eager bitblasting.